testnet/stacks-node/src/config.rs

```diff
@@ -398,14 +399,105 @@ lazy_static! {
 }
 
 impl Config {
-    /// This method applies any of this Config's configured PoX constants to the supplied
-    /// `PoxConstants` struct.
-    pub fn update_pox_constants(&self, pox_consts: &mut PoxConstants) {
-        if self.is_mainnet() {
+    /// Apply any test settings to this burnchain config struct
+    fn apply_test_settings(&self, burnchain: &mut Burnchain) {
+        if self.burnchain.get_bitcoin_network().1 == BitcoinNetworkType::Mainnet {
             return;
         }
-        if let Some(pox_2_activation_height) = self.burnchain.pox_2_activation {
-            pox_consts.v1_unlock_height = pox_2_activation_height;
+
+        if let Some(v1_unlock_height) = self.burnchain.pox_2_activation {
+            info!(
```

Comment:
In my testing environment where mining is configured for a 1 second interval, this log line is triggered around 100 times per second. Perhaps it would better as a `debug` level line?

Reply:
Fixed